Tasks/ShellScriptV2/task.json

```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 2,
         "Minor": 1,
-        "Patch": 4
+        "Patch": 5
```

Comment:
should we move this to using "minor is sprint" versioning we use for other tasks?

Reply:
Yeah, good idea